docs/i18n.rst

```diff
@@ -248,8 +237,6 @@ Typically, strings will be uploaded when a new release branch is cut from ``deve
 
 Before translators can begin working on the strings in our application, they need to be uploaded to Crowdin. Translations are maintained in release branches on Crowdin in the `Crowdin kolibri project <http://crowdin.com/project/kolibri>`__.
 
-.. warning:: You *must* have the Perseus plugin installed for development as described above, or not all strings will be extracted!
-
```

Comment:
Could add a note about updating and rebuilding the perseus repo to the frontend build pipeline docs:

https://kolibri-dev.readthedocs.io/en/develop/frontend_architecture/frontend_build_pipeline.html